index.js

```diff
@@ -1,31 +1,37 @@
 // using: regex, capture groups, and capture group variables.
-var templateUrlRegex = /templateUrl *:(.*)$/gm;
-var stylesRegex = /styleUrls *:(\s*\[[^\]]*?\])/g;
-var stringRegex = /(['"])((?:[^\\]\\\1|.)*?)\1/g;
+const componentRegex = /@Component\({([\s\S]*)}\)$/gm;
```

Comment:
you also want to provide a way for devs to include other decorators such as ionic `@App` or `@Page` and provide the previous regexp if they want it to be global